File: dart/lib/model/main_get_api_keys_resp.dart
part of swagger.api;

class MainGetApiKeysResp {
  
  String expiry = null;
  

  String label = null;
  

  String token = null;
  

  String tokenHash = null;
  
  MainGetApiKeysResp();

  @override
  String toString() {
    return 'MainGetApiKeysResp[expiry=$expiry, label=$label, token=$token, tokenHash=$tokenHash, ]';
  }

  MainGetApiKeysResp.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    expiry =
        json['expiry']
    ;
    label =
        json['label']
    ;
    token =
        json['token']
    ;
    tokenHash =
        json['tokenHash']
    ;
  }

  Map<String, dynamic> toJson() {
    return {
      'expiry': expiry,
      'label': label,
      'token': token,
      'tokenHash': tokenHash
     };
  }

  static List<MainGetApiKeysResp> listFromJson(List<dynamic> json) {
    return json == null ? new List<MainGetApiKeysResp>() : json.map((value) => new MainGetApiKeysResp.fromJson(value)).toList();
  }

  static Map<String, MainGetApiKeysResp> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MainGetApiKeysResp>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MainGetApiKeysResp.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/MainGetApiKeysResp.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace estuary-client.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class MainGetApiKeysResp {
    /// <summary>
    /// Gets or Sets Expiry
    /// </summary>
    [DataMember(Name="expiry", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "expiry")]
    public string Expiry { get; set; }

    /// <summary>
    /// Gets or Sets Label
    /// </summary>
    [DataMember(Name="label", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "label")]
    public string Label { get; set; }

    /// <summary>
    /// Gets or Sets Token
    /// </summary>
    [DataMember(Name="token", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "token")]
    public string Token { get; set; }

    /// <summary>
    /// Gets or Sets TokenHash
    /// </summary>
    [DataMember(Name="tokenHash", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "tokenHash")]
    public string TokenHash { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MainGetApiKeysResp {\n");
      sb.Append("  Expiry: ").Append(Expiry).Append("\n");
      sb.Append("  Label: ").Append(Label).Append("\n");
      sb.Append("  Token: ").Append(Token).Append("\n");
      sb.Append("  TokenHash: ").Append(TokenHash).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
